packages/openapi3/test/test-return-types.ts

```diff
@@ -353,13 +328,9 @@ describe("openapi3: return types", () => {
     `
     );
     strictEqual(diagnostics.length, 3);
```

Comment:
you can use the new `expectDiagnostics` util function to pass the list of diagnostic that it should expect